setup.py

```diff
@@ -22,6 +22,7 @@
     packages=find_packages(),
     license="BSD",
     entry_points={
+        "console_scripts": ["workflows-recipe-validate = workflows.validate:main"],
```

Comment:
I think something like `workflows.validate_recipe` is probably better? `workflows.` as common namespace prefix for all scripts and `validate_recipe` sounds more like a command?

Can you (whether you rename it or not) please add another entry point of the form
```python
  "libtbx.dispatcher.script": [ "workflows.validate_recipe = workflows.validate_recipe" ],
```
this is to add the command to dials environments with its original name, otherwise we would only get it as `libtbx.workflows.validate_recipe`.